test/ts-node-unit-tests/benchmarks/Grid/Grid-update.bench.ts

```diff
@@ -0,0 +1,58 @@
+import type { BenchmarkContext, BenchmarkResult } from '../../benchmark';
+import { performance } from 'node:perf_hooks';
+import { join } from 'node:path';
+import { setupDOM } from '../../test-utils';
+import { generateGridData } from '../../data-generators';
+
+export const config = {
+    sizes: [1_000, 10_000, 50_000, 100_000]
+};
+
+export function before(size: number) {
+    return {
+        fileName: `${size}-rows.json`,
```

Comment:
The file name of the previous benchmark is the same. I'm concerned about instability with the same row count. Probably should be different.